extensions/amp-sidebar/1.0/amp-sidebar-1.0.md

```diff
@@ -58,103 +58,232 @@ limitations under the License.
 - Touch zoom is disabled on the `amp-sidebar` and it's mask when the sidebar is open.
 
 Example:
+
 ```html
-<amp-sidebar id='sidebar1' layout='nodisplay'>
+<amp-sidebar id="sidebar1" layout="nodisplay" side="right">
   <ul>
     <li> Nav item 1</li>
     <li> Nav item 2</li>
     <li> Nav item 3</li>
     <li> Nav item 4</li>
     <li> Nav item 5</li>
     <li> Nav item 6</li>
-    <li> Nav item 7</li>
-    <li> Nav item 8</li>
-    <li> Nav item 9</li>
-    <li on="tap:sidebar1.close"> Close</li>
   </ul>
 </amp-sidebar>
 ```
 
-### Opening and Closing the Sidebar
-Setting the `on` attribute on one or more elements within the page and setting it's method to `toggle` will toggle the sidebar when the element is tapped or clicked. Setting the element's method to `open` or `close` will open or close the sidebar.Tapping back on the partially-visible main content area closes the sidebar.
+### Opening and closing the sidebar
+
+To toggle, open, or close the sidebar when an element is tapped or clicked, set the [`on`](../../../spec/amp-actions-and-events.md) action attribute on the element, and specify one of the following action methods:
+
+<table>
+  <tr>
+    <th>Action</th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td>open (default)</td>
+    <td>Opens the sidebar</td>
+  </tr>
+  <tr>
+    <td>close</td>
+    <td>Closes the sidebar</td>
+  </tr>
+  <tr>
+    <td>toggle</td>
+    <td>Toggles the sidebar state</td>
+  </tr>
+</table>
+
+If the user taps back on the partially-visible main content area, this closes the sidebar.
 
-Alternatively pressing the escape key on the keyboard will also close the sidebar.
+Alternatively, pressing the escape key on the keyboard will also close the sidebar.
 
 Example:
+
 ```html
 <button class="hamburger" on='tap:sidebar1.toggle'></button>
 <button on='tap:sidebar1'>Open</button>
 <button on='tap:sidebar1.open'>Open</button>
 <button on='tap:sidebar1.close'>x</button>
 ```
 
-## Attributes
-
-**side**
+### Toolbar
 
-The `side` attribute may be set to `left` or `right` depending upon whether sidebar should open in the left or right side of the page. If a `side` is not set on the `amp-sidebar` then it will be inherited from the `body` tag's `dir` attribute (`ltr` => `left` , `rtl` => `right`) and if one does not exist then the `side` is defaulted to `left`.
+Toolbar allows duplicating items within the sidebar, into the `<body>`, if the window matches a certain media query
 
-**open**
+#### Behavior
 
-The `open` attribute is present on the sidebar when it is open.
+- The sidebar may implement toolbars by adding nav elements with the toolbar attribute
```

Comment:
- Can the sidebar implement more than one toolbar? If not, change "toolbars" to "a toolbar".
- Use code style: `<nav>` and `toolbar`.
- Ending punctuation needed